src/Docusaurus/versioned_docs/version-v3/msbuild/authoring_first_msbuild_project.md

```diff
@@ -13,31 +13,31 @@ If you want to copy and paste this example, remember to change the &lt;ProjectGu
 value to match your own.
 
 ```
```

Comment:
If you make this:

   ```xml

Then we should get syntax highlighting in the XML. Since you're here. :)

Reply:
How we would get syntax highlighting? MD files doesn't support coloring

Reply:
This file is processed by Docusaurus, which uses MDX, not MD. You'll find other examples that have been updated to use the newly available XML fence.

Reply:
Can you provide some specific examples? 

All .md files I see either don't have formatting / colors or show the same bug I am correcting.

Reply:
https://github.com/wixtoolset/web/blob/master/src/Docusaurus/docs/intro.md

Reply:
Now fixed